### PR TITLE
simplify somewhat

### PR DESCRIPTION
**Pull request by derek-inclined**
Screen Shot 2021-09-15 at 2 59 58 PM

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "albion",
```
Comment (derek-inclined) [inline package-lock.json:8]: I don't know what this is...

File: pages/_document.tsx [hunk 2]
```diff
@@ -1,23 +0,0 @@
-import Document, { Html, Head, Main, NextScript } from 'next/document'
```
Comment (derek-inclined) [inline pages/_document.tsx:1]: This file isn't needed with no custom fonts.

File: pages/index.tsx [hunk 3]
```diff
@@ -10,69 +10,65 @@ export default function Home(): JSX.Element {
...
         <title>Loan Review</title>
       </Head>
-      <div className="h-screen bg-white overflow-hidden flex">
-        <div className="flex-1 w-0 flex flex-col">
-          <div className="md:px-16 md:py-32 overflow-y-auto">
-            <div className="max-w-prose flex flex-col sm:space-y-4">
-              <h2 className="inclined-text-5xl-roboto-black font-header">
-                Loans
-              </h2>
+      <main className='flex-1 overflow-y-auto focus:outline-none'>
```
Comment (derek-inclined) [inline pages/index.tsx:14]: I copied the layout from tailwindui.com.

**Comment by derek-inclined** [comment]
New screenshot:
Screen Shot 2021-09-15 at 3 53 07 PM

**Review by markshaw — APPROVED** [review]
LGTM

**Pull request merged**